app/assets/javascripts/discourse/app/components/topic-footer-buttons.js

```diff
@@ -46,9 +46,9 @@ export default Component.extend({
     return !isPM || this.canSendPms;
   },
 
-  @discourseComputed("currentUser.allowPersonalMessages")
+  @discourseComputed("currentUser.can_send_private_messages")
   canSendPms() {
-    return this.currentUser?.allowPersonalMessages;
+    return this.currentUser?.can_send_private_messages;
   },
```

Comment:
Actually this is a little interesting. Not sure when we decide to use aliases (like below) vs declaring a method like this.

Reply:
Actually yeah this can be an alias, will change